code/platformio_override.ini.example

```diff
@@ -0,0 +1,37 @@
+# -------------------------------------------------------------------------------------------------------------
+# Example file for 'platformio_override.ini' use.
+# Either copy and paste this file, then rename removing the .example or create your
+# own file: 'platformio_override.ini'
+# 'platformio_override.ini' allows users to create their own environments to add, remove or change
+# firmware configurations. 
+#
+# See: https://github.com/xoseperez/espurna/wiki/PlatformIO#customize-build-settings
+# for more details.
+# -------------------------------------------------------------------------------------------------------------
+
+// Example for a standard Sonoff Basic R2 or R3 but enabling DHT support on GPIO 2.

```

Comment:
afaik, there are only 2 allowed comment prefixes - `;` and `#`. I don't think `//` will work

Implementation detail:
https://docs.python.org/3/library/configparser.html#customizing-parser-behaviour
> comment_prefixes, default value: ('#', ';')

https://github.com/platformio/platformio-core/blob/3ef96cb215024709f463d5b1b5574ad66bca3ebf/platformio/project/config.py#L70 for value lists
https://github.com/platformio/platformio-core/blob/3ef96cb215024709f463d5b1b5574ad66bca3ebf/platformio/project/config.py#L94 for other stuff

Reply:
I have got my comment prefixes muddled between the `.h` and `.ini` files. :-)